client/test-data.cpp

```diff
@@ -187,75 +157,53 @@ namespace tec {
 		}
 	}
 
-	// NOTE : This would be change on a future, not ? Actually is loading/saving from assets (where not is supposed to be write by a user)
+	std::string LoadJSON(const FilePath& fname) {
+		std::fstream input(fname.GetNativePath(), std::ios::in | std::ios::binary);
+		std::string in;
+		input.seekg(0, std::ios::end);
+		in.reserve(input.tellg());
+		input.seekg(0, std::ios::beg);
+		std::copy((std::istreambuf_iterator<char>(input)), std::istreambuf_iterator<char>(), std::back_inserter(in));
+		input.close();
+		return std::move(in);
```

Comment:
I think that this move is now mandated by the standard even if you do `return in;`.